scripts/migrations/0_1_7_up.ts

```diff
@@ -0,0 +1,44 @@
+import { types as T } from "../deps.ts";
+import {
+  matchFilebrowserHomepageConfig,
+  matchFilebrowserSubdomain,
+  matchSubdomains,
+  matchWebPageHomepage,
+  matchWebPageHomepageConfig,
+  matchWebPageSubdomain,
+} from "./types.ts";
+
+export const convertHomepageConfig = (config: T.Config) => {
+  if (matchFilebrowserHomepageConfig.test(config)) {
+    const newHomepage: typeof matchWebPageHomepage._TYPE = {
+      type: "web-page",
+      source: "filebrowser",
+      folder: config.homepage.directory!,
+    };
+    (config as typeof matchWebPageHomepageConfig._TYPE).homepage = newHomepage;
```

Comment:
A. We could export the types instead of doing this on each of the page
B. I would do instead 
```ts
return {
   ...config,
   homepage: newHomepage
}
```